security/transport/transaction.py

```diff
@@ -80,3 +82,40 @@ def is_active_logged_requests(using=None):
     :return: True if block of code is surrounded with atomic_log operator
     """
     return hasattr(get_connection(using), 'output_logged_requests') and get_connection(using).output_logged_requests
+
+
+
+def get_all_request_related_objects():
+    """
+    Return all related selected with log_request_related_objects decorator.
+    :return: list of request related objects
+    """
+    connection = get_connection()
```

Comment:
Huh, that's interesting solution - using DB connection to store request params :smile:

Reply:
yes I found it in the django implementation :D